models/ConfigureForm.php

```diff
@@ -83,6 +90,26 @@ public function saveSettings()
         return true;
     }
 
+    public function validateAge()
```

Comment:
Honestly I'm not really sure if I got this right or not :sweat_smile:

Reply:
Maybe something like this would be easier?
```php
    public function validateAge()
    {
        if (!empty($this->defaultAge)) {
            return;
        }

        preg_match_all(static::REGEX_NUMBERS, $this->defaultAge, $matches, PREG_SET_ORDER);
        if(!isset($matches[0][0])) {
            $this->addError('defaultAge', Yii::t('LegalModule.base', 'Invalid Age!'));
        } else {
            $this->defaultAge = $matches[0][0];
        }
    }
```

Reply:
As I said before I think the validation can be simplified, which validation do you want here?

Reply:
Generally speaking I was wanting to validate an age number between 0 and 99 or at least that was the general idea of this 🤔

Reply:
Remove the `validAge()` and replace the validation rule with:

```
['defaultAge', 'integer', 'min' => 0, 'max' => 99];
```

Reply:
Done